src/easier68k/core/models/list_file.py

```diff
@@ -0,0 +1,77 @@
+"""
+List File
+
+Represents the output from the assembler that contains all of the instructions
+"""
+
+
+class ListFile:
+    def __init__(self):
+        """
+        Construcctor
+
+        The raw assembly instructions
+        """
+        self.content = bytearray([0x00])
+
+    def load_bytes(self, bytes: bytearray) -> bytearray:
+        """
+        Loads the contents of a byte array
+        into the contents of this list file
+        :param bytes:
+        :return:
+        """
+        self.content = bytes
+
+    def load_from_str(self, bytestr: str) -> str:
+        """
+        Loads the contents of a string into the contents
+        of this list file.
+        :param bytestr:
+        :return:
+        """
+        self.content = bytestr.encode('ascii')
+
+    def load_easy68k_list_file(self, contents: str) -> str:

```

Comment:
If this method is for easy68k list files, does that mean that `load_from_str` is designed for raw hex (not list files)? In which case, should this be in the list file class?